web-common/src/features/canvas/components/table/selector.ts

```diff
@@ -130,23 +132,25 @@ export function getTableConfig(
   );
 }
 
-export function hasValidTableSchema(
-  instanceId: string,
-  tableProperties: TableProperties,
-) {
+export function validateTableSchema(
+  ctx: StateManagers,
```

Comment:
Ideally this weren't coupled to a Canvas `ctx` and rather accepted a `metricsView`